core/src/com/unciv/ui/worldscreen/WorldScreen.kt

```diff
@@ -175,8 +181,68 @@ class WorldScreen(val gameInfo: GameInfo, val viewingCiv:CivilizationInfo) : Cam
     }
 
     private fun addKeyboardPresses() {
+        val quickSave = {
+            val toast = ToastPopup("Quicksaving...", this)
+            thread(name = "SaveGame") {
+                GameSaver.saveGame(gameInfo, "QuickSave") {
+                    Gdx.app.postRunnable {
+                        toast.close()
+                        if (it != null)
+                            ToastPopup("Quicksave failed!", this)
+                        else {
+                            ToastPopup("Quicksave OK.", this)
+                            UncivGame.Current.setWorldScreen()
+                        }
+                    }
+                }
+            }
+            Unit
```

Comment:
???

Reply:
Kludge - thread() returns a Thread instance so this casts the return type of the anonymous function to void - ahem, Unit as the dispatcher expects ()->Unit not ()->Thread. There's many alternate solutions, this is maybe the shortest that incurs no additional overhead - but probably you're right it's not obvious. Should the code stay at all I'd simply comment it. An explicit `as` cast would be unchecked...